test/unit/ajax.js

```diff
@@ -1408,6 +1408,103 @@ QUnit.module( "ajax", {
 		};
 	} );
 
+	ajaxTest( "jQuery.ajax() - responseURL", 13, function( assert ) {
+		function expectedUrl( url ) {
+			return jQuery.support.responseURL ? url : "";
+		}
+
+		/**
+		 * this function generates complete
```

Comment:
Please start sentences with an uppercase letter. Also, we usually use single-line comments (`//`) even for comments spanning multiple lines.